### ISSUE TITLE
"collection not opened"

### ISSUE THREAD

**Issue by hamidietly** [ankidroid/Anki-Android#6580]
Edited to fix the title 

###### Reproduction Steps

1. Sync my decks
2. Sync conflict (intentionally)
3. Download and overwrite
4. "Collection could not be opened. An error occured while accessing your collection. It may be caused by a harmless bug, or there may be a problem with it" pop up.

I can get it operational if I follow prompts of "options --> check database" but I do not think my progress is being synchronised across devices and I'm worried if I add a card on my phone I won't see it on the computer or tablet.

###### Expected Result

Successful synching.

###### Actual Result
Listed above.

###### Debug info
AnkiDroid Version = 2.11.3

Android Version = 10

ACRA UUID = 51680dac-3938-4f1f-a2ee-d90795a33948

////////////////Additional info//////////

- I've tried the "full sync from server" button on the error handling but the problem just restarts all over

- I went to directory and manually deleted all backups and collections and everything in the Anki folder, erased app data and cache and redownloaded all off the decks again using the apps account sync and got the same issue 

- everything seems fine on my Mac version and on my iPad Anki app. (Mac version is 2.1.26 and MacOS is 10.15.5, iPad app is 2.063 (20063.6)

###### Research
*Enter an [x] character to confirm the points below:*

- [ x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum ----- there was one similar recent case but it was not answered at the time of writing this and I thought maybe my case could help the team figure this out, if this is not in line with procedures and rules I apologize. 
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by mikehardy** [comment]
Hi there! Can you verify if this happens with the latest alphas? You can download a "parallel" version just to investigate (for instance, choose the '.A' .apk and it installs as 'com.ichi2.anki.AnkiDroid.A' right next to your main version) https://github.com/ankidroid/Anki-Android/releases

Also, if you sync with desktop, knowing your desktop version would help, we've had a few incompatibilities come up recently with folks using the latest desktop code - the pace of change there is pretty high in the desktop internals right now and it occasionally shows up as tiny unexpected mismatches between what data desktop produces and what we expect, with issues like this being where we realize for the first time it happened and work together to get in sync again

**Comment by hamidietly** [comment]
Hello Mark,

So I tried it on both a and b parallel versions and I got the same issue on both but I can see at the bottom the number of cards studied and numbers of minutes are correct when I got "options" --> check database.

Mac desktop version is 2.1.26 and MacOS is 10.15.5.

Let me know if you need any further info! :)

And sorry for the delayed response, I had tried paying here but it kept telling me I could "not respond at this time, please try again later" so I replied to the email thinking it would function to post a reply as well but it did not.

**Comment by BenBergman** [comment]
I'm getting the same error message after deleting a deck on desktop and importing the latest version (Ultimate Geography). Part of that process recommended deleting the Ultimate Geography card type and then syncing, so I'm wondering if that might be related in my case (let me know if I should open my own ticket).

I'm running Anki 2.1.26 on Arch Linux and 2.11.3 on Android 10.

**Comment by Arthur-Milchior** [comment]
How can I access reports ? I see that acra is https://github.com/ankidroid/Anki-Android/issues/6580 and have access to ACRA, but can't figure out how to search using a particular number

**Comment by david-allison-1** [comment]
@Arthur-Milchior 

I've added a custom search engine in Chrome for it.

`https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/reports-browser/user/%s`

https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/reports-browser/user/51680dac-3938-4f1f-a2ee-d90795a33948

**Comment by david-allison-1** [comment]
@BenBergman @hamidietly 

Would you be willing to share your Anki Database (`collection.anki2`) file so we can further debug this issue?

 It includes all cards, decks and review history, but not media.

This can be uploaded to cloud storage and linked in a comment.

Alternately, this can be sent privately to me at gmail: davidallisongithub

All the best,

David

**Comment by Arthur-Milchior** [comment]
@BenBergman First, let me tell you I admire you for being able to run an Arch Linux

This error message is quite general, so it may be related but not necessarily. Furthermore, when opening a bug report, it'll tell you what information we want, such as the id which allows us to find the crash report, which is usually useful for debugging. (Or you can look for the crash in logcat)

@BenBergman @hamidietly Did you accept that AnkiDroid sends data to us in case of problem ? If no, it may explain why we didn't find any result I assume

**Comment by hamidietly** [comment]
Hey! Sorry for the delay!
https://www.dropbox.com/sh/e22q0vc44klc8x1/AAASmo3cMAl4MzBLYn72reoma?dl=0 this is the Anki collection files as requested. I think I have that now listed accepted so perhaps it will show now?

**Comment by hamidietly** [comment]
AnkiDroid Version = 2.11.3

Android Version = 10

ACRA UUID = 51680dac-3938-4f1f-a2ee-d90795a33948

In case the ACRA may have changed

**Comment by david-allison-1** [comment]
@hamidietly The collection works fine on my device.

If you sync after fixing the collection, then the fix will be propagated to Anki Desktop. If you download again, then the fix will be reverted.

**Comment by BenBergman** [comment]
@Arthur-Milchior Arch isn't too hard once you are used to it. :)

Thanks for the info. I just globbed onto an existing issue instead of reading up on the proper flow. I will keep that in mind for the next one.

@david-allison-1 Doing another sync in AnkiDroid and selecting upload, then opening Anki on the desktop and selecting download did the trick. I was a bit nervous everything would be wiped out, but I did an export on desktop as a fail safe. All good now!
